.cookiecutter/cookiecutter.json

```diff
@@ -7,6 +7,7 @@
     "lms/app.py",
     "tests/unit/lms/app_test.py",
     "lms/pshell.py",
+    "lms/migrations/env.py",
```

Comment:
LMS seems to have some of its own customizations to `env.py`. Perhaps one day we can remove `env.py` from LMS's cookiecutter ignore list and migrate it to the standard `env.py` from the cookiecutter. But at least for now I don't want to deal with it.